lib/ansiblelint/cli.py

```diff
@@ -55,8 +55,13 @@ def load_config(config_file):
     try:
         with open(config_path, "r") as stream:
             config = yaml.safe_load(stream)
-    except yaml.YAMLError:
-        return None
+    except yaml.YAMLError as e:
+        print(e, file=sys.stderr)
+        sys.exit(2)
+    # TODO(ssbarnea): implement schema validation for config file
+    if isinstance(config, list):
+        print("Invalid configuration '%s', expected dictionary." % config_path, file=sys.stderr)
```

Comment:
It's called a mapping in terms of YAML. Also, it's better to use `str.format()` for interpolation.
```suggestion
        print(
        	"Invalid configuration '{cfg!s}', expected YAML mapping in the config file.".
        	format(cfg=config_path),
        	file=sys.stderr,
        )
```